es/indexer.go

```diff
@@ -268,17 +288,49 @@ func UpdateSynonyms(esc *elastic.Client, indexNameByLang IndexNameByLang) error
 
 		defer openIndex(indexName, esc)
 
-		settingsRes, err := esc.IndexPutSettings(indexName).BodyJson(body).Do(context.TODO())
+		bodyStr, err := json.Marshal(body)
+		if err != nil {
+			return errors.New(fmt.Sprintf("Failed marshding %+v.", body))
+		}
+		url := fmt.Sprintf("%s/%s/_settings", viper.GetString("elasticsearch.url"), indexName)
+		log.Infof("Sending to %s: %s", url, string(bodyStr))
+		contents, err := putRequest(url, bytes.NewBuffer(bodyStr))
```

Comment:
Add comment:
# Using standard http put call instead of esc.IndexPutSettings(indexName).BodyJson(body).Do(context.TODO()) due to the fact that this version of elastic hides error when synonyms are not updated properly.

Reply:
added